packages/dendron-next-server/pages/vscode/seed-browser.tsx

```diff
@@ -0,0 +1,137 @@
+
+  if (status === "loading") {
+    return <div>Loading styles...</div>;
+  }
+
+  const seedDataToRender = _.values(seedRegistryData).map((data) => {
+    if (data) {
+      let payload: SeedConfig & { seedInWorkspace: boolean } = {
+        ...data,
+        seedInWorkspace: false,
+      };
+      if (_.includes(ide.seedsInWorkspace, data?.id)) {
+        payload = {
+          ...payload,
+          seedInWorkspace: true,
+        };
+      }
+
+      return payload;
+    }
+  });
+
+  const toggleDarkMode = () => {
+    setIsDarkMode((previous) => {
+      switcher({ theme: previous ? themes.light : themes.dark });
+      return !previous;
+    });
+  };
+
+  const { Meta } = Card;
+  const { Footer, Content } = Layout;
+
+  return (
+    <>
+      <Layout className={seedStyles.layout}>
+        <Content style={{ padding: "0 50px" }}>
+          <div className={seedStyles.contentDiv}>
+            <PageHeader
+              className={seedStyles.siteHeader}
+              title="Dendron Seed Registry"
+              subTitle="Add Knowledge Bases to your Workspace"
+            />
+            <div className={seedStyles.listDiv}>
+              <List
+                grid={{
+                  gutter: 24,
+                  xs: 1,
+                  sm: 2,
+                  md: 3,
+                  lg: 3,
+                  xl: 4,
+                  xxl: 4,
+                }}
+                dataSource={seedDataToRender}
+                renderItem={(item) => (
+                  <List.Item>
+                    <Card
+                      className={seedStyles.card}
+                      hoverable
+                      actions={[
+                        <GoToSiteButton
+                          url={item!.site ? item!.site.url : undefined}
+                          inVscode={!browser}
+                        />,
+                        <AddToWorkspaceButton
+                          seedId={item?.id!}
+                          existsInWorkspace={
+                            item ? item.seedInWorkspace : false
+                          }
+                        />,
+                      ]}
+                    >
+                      <Meta
+                        className={seedStyles.cardMeta}
+                        avatar={
+                          <Avatar
+                            src={
+                              item?.assets
+                                ? item.assets.publisherLogo
+                                : undefined
+                            }
+                          />
+                        }
+                        title={item!.name}
+                        description={item!.description}
+                      />
+                    </Card>
+                  </List.Item>
+                )}
+              />
+            </div>
+          </div>
+        </Content>
+        <Footer>
+          <h4>Current theme: {currentTheme}</h4>
```

Comment:
can we remove this? since we are going to support custom theming, this would introduce extra complexity in terms of keeping track of per view themes vs global themes

Reply:
Yes, good catch - this was meant to be removed.  There are a few more UI elements that I need to clean up.